app.py

```diff
@@ -60,16 +63,26 @@ class Stop:
   name: str
   predictions: typing.List[Prediction]
 
+@dataclass
+class Cache:
+  stops: typing.List[Stop]
+  updated_at: str
+
 PREDICTIONS_URL = 'https://api.511.org/transit/StopMonitoring'
-cache = []
+cache = Cache([], '')
 def update_cache():
-    # clear cache of any previous (now outdated) data
-    cache.clear()
+    agency_to_stop_suffix = {
+        "BA": "BART",
+    }
+    new_cache = []
```

Comment:
`new_cache` poorly explains the use of this list, `new_stops` would probably be better suited

Reply:
fixed